### ISSUE TITLE
Technical Decision-Making: Discussing "various factors outlined on next page" too vague

### ISSUE THREAD

**Issue by utako** [pivotal-cf/areas-of-contribution#5]
### P2: Surfaces and discusses various factors (outlined in P3+) when approaching a technical decision, with support from other pivots to make the final decision. 

This has added cognitive overhead to my feedback collection, as the next page is not visible. We should either make the factors explicit on this page or reword this to be more accurate.

**Comment by rosenhouse** [comment]
Thanks for opening this.

As I mentioned in your other issue, we're starting to re-invest in this repo, and the first priority is to enable a pull-request-based contribution model for changing the content here.  We've put a bit more detail about our plans [this roadmap](https://github.com/pivotal-cf/areas-of-contribution/blob/master/CONTRIBUTING.md#roadmap).

We don't have anything actionable yet. But once we've made concrete progress on that roadmap, we intend to respond again here with pointers to a contribution workflow.

Thanks for your patience!
